### DISCUSSION TITLE
2008/09/21/vi-cup-%E8%AE%BE%E8%AE%A1%E7%AB%9E%E8%B5%9B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#244] [Announcements]
# 2008/09/21/vi-cup-%E8%AE%BE%E8%AE%A1%E7%AB%9E%E8%B5%9B

最近公司组织了一次由在校大学生参加的电子设计竞赛，叫做 VI Cup。今天是竞赛的决赛日，我很荣幸成为了软件设计方面的评委之一。我作为研发工程师，平时与公司其它部门的同事，乃至公司之外的用户、学生等接触的都比较少，所以一直不是很全面的了解其他部门同事和客户的工作。这次活动让我让我在这方面有了新的认识。

https://blog.qizhen.xyz/2008/09/21/vi-cup-%E8%AE%BE%E8%AE%A1%E7%AB%9E%E8%B5%9B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

L
2008年12月16日 15:18 编辑
luguo

回复
Qizhen
2008年09月26日 11:03 编辑
我平时不大用MSN聊天，你可以发Email给我，ruanqizhen@gmail.com

回复
崴
2008年09月25日 05:51 编辑
hi 你好 我在google时找到了永青的blog，然后连到你这儿来，发现你是LabVIEW中国的牛人！我原来的几个project都是用LabVIEW做的，对它非常感兴趣，甚至可以说有种情感吧！我今年硕士毕业，对Austin的职位Application Engineer，就是ELP那个项目坚定了信心去拿下，结果在招聘会上知道，今年开始NI总部不帮学生办工作签证，我投了简历，还参加了info seminar，结果连面试机会也不给因为我觉得在哪都是学，而且我想NI在中国的市场增长率比美国肯定要大，可以得到更多的锻炼机会，所以我很想加入到你们的团队我在msn上发了邀请了，希望你有空时我们能聊聊，谢谢！

回复
翟
2008年09月22日 18:09 编辑
如果我们后辈能被你们夸奖，那也是多拜你们这些前辈所赐啊~~

回复
奕瑋
2008年09月22日 16:18 编辑
現今的學生們，但凡努力的都很不錯，感覺上他們的理解力和智商都比我們高。
真是長江后浪推前浪啊。

回复
arrowpig
2008年09月21日 21:30 编辑
你可已经是国内LabVIEW的专家啊

回复
